homeassistant/components/usb/__init__.py

```diff
@@ -136,3 +129,18 @@ def _async_process_ports(self, ports: list[ListPortInfo]) -> None:
     def scan_serial(self) -> None:
         """Scan serial ports."""
         self.hass.add_job(self._async_process_ports, comports())
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command({vol.Required("type"): "usb/scan"})
+@websocket_api.async_response
+async def websocket_usb_scan(
+    hass: HomeAssistant,
+    connection: ActiveConnection,
+    msg: dict,
+) -> None:
+    """Scan for new usb devices."""
+    usb_discovery: USBDiscovery = hass.data[DOMAIN]
+    if not usb_discovery.observer_active:
+        await hass.async_add_executor_job(usb_discovery.scan_serial)
```

Comment:
Need 1 minute debounce here so they can't hit it too often

Reply:
We have the debouncer helper.

https://github.com/home-assistant/core/blob/debc6d632c7e5025c516968e1e3adb9ccdbc43d8/homeassistant/helpers/debounce.py#L12

Reply:
perfect.  Will pick this back up after I finish up https://github.com/home-assistant/core/pull/54938#issuecomment-902955930